includes/Checker/Checks/Abstract_PHP_CodeSniffer_Check.php

```diff
@@ -75,6 +75,23 @@ final public function run( Check_Result $result ) {
 		// Backup the original command line arguments.
 		$orig_cmd_args = $_SERVER['argv'];
 
+		if ( class_exists( '\PHP_CodeSniffer\Config' ) ) {
+			/*
+			 * Reset \PHP_CodeSniffer\Config::$overriddenDefaults to prevent
+			 * incorrect results when running multiple checks.
+			 *
+			 * PHPStan ignore reason: PHPStan raised an issue because we can't
+			 * use class in ReflectionClass.
+			 *
+			 * @phpstan-ignore-next-line
+			 */
+			$reflected_phpcs_config = new \ReflectionClass( '\PHP_CodeSniffer\Config' );
+			$overridden_defaults    = $reflected_phpcs_config->getProperty( 'overriddenDefaults' );
+			$overridden_defaults->setAccessible( true );
+			$overridden_defaults->setValue( array() );
+			$overridden_defaults->setAccessible( false );
+		}
```

Comment:
Personal preference, I would have this logic in a protected method.

Reply:
@mukeshpanchal27 See above, however it should be a `private` method since there's no reason at this point to expose it to child classes. Can you move this to a method please?

Reply:
@mukeshpanchal27 +1 on the above, lets move this to a `private` method.

Reply:
Thanks for the feedback. In https://github.com/10up/plugin-check/pull/214/commits/55a8867d363dbeff1572bb29c2d9dba939e4f8ab i have move the code in `private` mathod.